src/js/providers/html5.ts

```diff
@@ -444,6 +451,14 @@ function VideoProvider(this: HTML5Provider, _playerId: string, _playerConfig: Ge
         });
     };
 
+    function _canPlayHandler(): void {
+        _canSeek = true;
+        if (!_androidHls) {
+            _setMediaType();
+        }
+        checkVisualQuality();
```

Comment:
`_setMediaType` and `checkVisualQuality` should not be deferred until "canplaythrough".